tests/TestGui/run_project_test/run_project.tcl

```diff
@@ -17,5 +17,5 @@
 #You should have received a copy of the GNU General Public License
 #along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-puts "OPENING PROJECT"; flush stdout ; open_project test.ospr
-puts "PROJECT OPENED"; flush stdout ; exit
+puts "RUNNING PROJECT"; flush stdout ; run_project test.ospr
```

Comment:
Error out at failure.
Refer: https://github.com/os-fpga/FOEDAG/blob/main/tests/TestBatch/test_ip_generate.tcl#L42

Reply:
I did it in a bit another way but the idea is similar: if foedag.log contains 'failed', we exit with an error. output PASSED otherwise:
![image](https://user-images.githubusercontent.com/109239890/199088500-0474a5a4-14c9-4e06-91b2-c7748ccbfaf8.png)

Reply:
We discussed about adding it the way test_ip_generate.tcl was implemented, hence the comment pointing to it. Why can't we use that approach? The grep way to check is external to the tcl. I would like to have the check built as part of the test tcl itself.

Reply:
fixed. verification was moved from Makefile to the tcl script